Sources/QueuesRedisDriver/JobsRedisDriver.swift

```diff
@@ -55,6 +57,26 @@ public struct RedisQueuesDriver {
     ///   - eventLoopGroup: The `EventLoopGroup` to run the driver with
     public init(configuration config: RedisConfiguration, on eventLoopGroup: EventLoopGroup) {
         let logger = Logger(label: "codes.vapor.redis")
+        let eventLoop = eventLoopGroup.next()
+        let redisTLSClient: ClientBootstrap? = {
+            guard let tlsConfig = config.tlsConfiguration,
+                    let tlsHost = config.tlsHostname else { return nil }
+
+            return ClientBootstrap(group: eventLoop)
+                .channelOption(ChannelOptions.socket(SocketOptionLevel(SOL_SOCKET), SO_REUSEADDR), value: 1)
```

Comment:
The `SocketOptionName` and `SocketOptionValue` types should also be explicitly invoked here to ensure the build succeeds on all platforms.

```suggestion
                .channelOption(ChannelOptions.socket(SocketOptionLevel(SOL_SOCKET), SocketOptionName(SO_REUSEADDR)), value: SocketOptionValue(1))
```